src/Parsers/obfuscateQueries.cpp

```diff
@@ -24,413 +27,50 @@ namespace ErrorCodes
-    "MEMORY",
-    "MI",
-    "MICROSECOND",
-    "MILLISECOND",
-    "MIN",
-    "MINUS",
-    "MINUTE",
-    "MM",
-    "MOD",
-    "MODIFY",
-    "MONTH",
-    "MOVE",
-    "MS",
-    "MULTIIF",
-    "MUTATION",
-    "NAME",
-    "NAMED",
-    "NANOSECOND",
-    "NEXT",
-    "NO",
-    "NONE",
-    "NOT",
-    "NOTEQUALS",
-    "NOTIN",
-    "NS",
-    "NULL",
-    "NULLS",
-    "OBJECT",
-    "OFFSET",
-    "ON",
-    "ONLY",
-    "OPTIMIZE",
-    "OPTION",
-    "OR",
-    "ORDER",
-    "OUTER",
-    "OUTFILE",
-    "OVER",
-    "OVERRIDE",
-    "PART",
-    "PARTIAL",
-    "PARTITION",
-    "PARTITIONS",
-    "PART_MOVE_TO_SHARD",
-    "PERMANENTLY",
-    "PERMISSIVE",
-    "PIPELINE",
-    "PLAN",
-    "PLUS",
-    "POLICY",
-    "POPULATE",
-    "POSITION",
-    "PRECEDING",
-    "PRECISION",
-    "PREWHERE",
-    "PRIMARY",
-    "PRIVILEGES",
-    "PROCESSLIST",
-    "PROFILE",
-    "PROJECTION",
-    "QQ",
-    "QUARTER",
-    "QUERY",
-    "QUOTA",
-    "RANDOMIZED",
-    "RANGE",
-    "READONLY",
-    "REALM",
-    "RECOMPRESS",
-    "REFERENCES",
-    "REFRESH",
-    "REGEXP",
-    "REGEXPQUOTEMETA",
-    "REMOVE",
-    "RENAME",
-    "REPLACE",
-    "REPLACEREGEXPALL",
-    "REPLACEREGEXPONE",
-    "RESET",
-    "RESTORE",
-    "RESTRICT",
-    "RESTRICTIVE",
-    "RESUME",
-    "REVOKE",
-    "RIGHT",
-    "ROLE",
-    "ROLES",
-    "ROLLBACK",
-    "ROLLUP",
-    "ROW",
-    "ROWS",
-    "RTRIM",
-    "S3",
-    "SALT",
-    "SAMPLE",
-    "SECOND",
-    "SELECT",
-    "SEMI",
-    "SERVER",
-    "SET",
-    "SETS",
-    "SETTING",
-    "SETTINGS",
-    "SHA256_HASH",
-    "SHARD",
-    "SHOW",
-    "SIGNED",
-    "SIMPLE",
-    "SINGLEVALUEORNULL",
-    "SNAPSHOT",
-    "SOURCE",
-    "SPATIAL",
-    "SS",
-    "STDOUT",
-    "STEP",
-    "STORAGE",
-    "STRICT",
-    "STRICTLY_ASCENDING",
-    "SUBPARTITION",
-    "SUBPARTITIONS",
-    "SUBSTRING",
-    "SUSPEND",
-    "SYNC",
-    "SYNTAX",
-    "SYSTEM",
-    "TABLE",
-    "TABLES",
-    "TEMPORARY",
-    "TEST",
-    "THAN",
-    "THEN",
-    "TIES",
-    "TIMESTAMP",
-    "TIMESTAMP_ADD",
-    "TIMESTAMPADD",
-    "TIMESTAMP_DIFF",
-    "TIMESTAMPDIFF",
-    "TIMESTAMP_SUB",
-    "TIMESTAMPSUB",
-    "TO",
-    "TODATE",
-    "TODATETIME",
-    "TOP",
-    "TOTALS",
-    "TRACKING",
-    "TRAILING",
-    "TRANSACTION",
-    "TREE",
-    "TRIGGER",
-    "TRIM",
-    "TRIMBOTH",
-    "TRIMLEFT",
-    "TRIMRIGHT",
-    "TRUE",
-    "TRUNCATE",
-    "TTL",
-    "TUPLE",
-    "TYPE",
-    "UNBOUNDED",
-    "UNFREEZE",
-    "UNION",
-    "UNIQUE",
-    "UNSIGNED",
-    "UNTUPLE",
-    "UPDATE",
-    "URL",
-    "USE",
-    "USER",
-    "USING",
-    "UUID",
-    "VALUES",
-    "VARYING",
-    "VIEW",
-    "VIEWIFPERMITTED",
-    "VISIBLE",
-    "VOLUME",
-    "WATCH",
-    "WATERMARK",
-    "WEEK",
-    "WHEN",
-    "WHERE",
-    "WINDOW",
-    "WITH",
-    "WK",
-    "WRITABLE",
-    "YEAR",
-    "YYYY",
-    "ZKPATH"
+    static std::unordered_set<std::string_view> instance;
```

Comment:
maybe

```suggestion
    static std::unordered_set<std::string_view> instance = initialize();
```

and remove `initialized`